### DISCUSSION TITLE
Blazor.Bootstrap/Blazor.Bootstrap.bundle.scp.css missing after update from version 3.1.1 to 3.2.0

### DISCUSSION THREAD

**Discussion by laboratoriumkodu** [vikramlearning/blazorbootstrap#1039] [Q&A]
After updates Nuget package from version  3.1.1 to 3.2.0 program stopped generate Blazor.Bootstrap.bundle.scp.css. Before update and after update revoked back to version 3.1.1 everything works fine. Project's configuration on .net8 as interactive web assembly. All other library are  up to date.

**Comment by gvreddy04** [comment]
@laboratoriumkodu Thank you for using BlazorBootstrap. Please take a look at our starter templates.

Starter Templates: https://github.com/vikramlearning/blazorbootstrap-starter-templates
